management/global/sso/policies.tf

```diff
@@ -222,7 +223,9 @@ data "aws_iam_policy_document" "data_scientist" {
     actions = [
       "athena:*",
       "apigateway:*",
+      "aoss:*",
       "autoscaling:*",
+      "aws-marketplace:*",
```

Comment:
@exequielrafaela would you help me understand why this permission is needed? And why so permissive also?